docker-wrappers/OmicsIntegrator1/Dockerfile

```diff
@@ -24,7 +22,5 @@ RUN commit=0a57ede6beeef6e63b86d19898e560d62015e85d && \
 ENV MSGSTEINER_PATH=/OmicsIntegrator/msgsteiner-1.3/msgsteiner
 WORKDIR /OmicsIntegrator
 
-COPY environment.yml .
```

Comment:
We could delete `environment.yml` if we are no longer using it to build the image in either Dockerfile.